config/formats.ts

```diff
@@ -1643,6 +1643,21 @@ export const Formats: FormatList = [
 			// Shadow Tag/Arena Trap
 			'Diglett-Base', 'Dugtrio-Base', 'Gothita', 'Gothitelle', 'Gothorita', 'Trapinch', 'Wobbuffet', 'Wynaut',
 		],
+		onValidateSet(set) {
```

Comment:
The third parameter is actually the `setHas` variable, so it seems as if you could actually move the code from the team validator block to this function. (But I haven't actually tried it to see whether it works.)